go/vt/wrangler/schema.go

```diff
@@ -108,8 +108,7 @@ func (wr *Wrangler) ValidateSchemaShard(ctx context.Context, keyspace, shard str
 	return nil
 }
 
-// ValidateSchemaKeyspace will diff the schema from all the tablets in
-// the keyspace.
+// ValidateSchemaKeyspace will diff the schema from all the tablets in the keyspace.
```

Comment:
can we reimplement this using the new apis?

Reply:
Oh yeah! Forgot to migrate this one.

Reply:
Just so I understand correctly, for migration, we need to change both the vtctl command (https://github.com/vitessio/vitess/pull/9667/files#diff-ad37010accc5a29c6751d18124328bb882457e8b2527db688b854c81fe23861cR3222)  and change the replaced wrangler function?

Reply:
Yeah, so `wrangler` can import an instance of `grpcvtctldserver`, and can then call the "new" method instead of having its own implementation. Similar to [this one](https://github.com/vitessio/vitess/blob/18ea5887823a33c642759b92d61b3c08233fbc35/go/vt/wrangler/keyspace.go#L57-L67)

Reply:
Sounds good! 😄 I'll replace both the methods! (ValidateVersion/ValidateSchema)